src/components/attendee/Popover.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import ReactDOM from "react-dom";
+import { usePopper } from "react-popper";
+
+import { POPOVER_CONTAINER_ID } from "settings";
+
+export interface PopoverProps {
+  referenceElement?: Element | null;
+}
+
+export const Popover: React.FC<PopoverProps> = ({
+  children,
+  referenceElement,
+}) => {
+  const popoverContainerElement = document.querySelector(
+    `#${POPOVER_CONTAINER_ID}`
+  );
```

Comment:
[Q] Why isn't the element itself passed instead (through ref maybe)? 

[NOTE] We can probably use the new global object `SPARK` to share few of these references. Might be OOS to this PR, but it seems like a tidying up we can do later on.

Reply:
because the element `#${POPOVER_CONTAINER_ID}` is located not as a parent node, but somewhere as an independent node in the main layout, so when we don't have the reference to the  `#${POPOVER_CONTAINER_ID}` in the place where we use `<Popover />`.

It's needed for proper z-index layout.

here's the approximate schema:

```html
<main>
 <AttendeeContent>
 ....
      <PortalComponent>
          <Popover>
            ... looking for the #POPOVER_CONTAINER_ID node to put the content of the modal
           </Popover>
      </PortalComponent>
 .... 
 <AttendeeContent>
 
 <div id={POPOVER_CONTAINER_ID}></>
</main>
```